### DISCUSSION TITLE
B站最近又再作妖了，能治一治自动连播吗？

### DISCUSSION THREAD

**Discussion by sherlocknieh** [the1812/Bilibili-Evolved#1740] [功能建议 / Ideas]
以前单个视频【分P自动连播】和【推荐视频的连播】是分开的
现在被合并了，非常烦人，求大神治一治

**Comment by Geeyun-JY3** [comment]
下次开 discussion 前请先善用搜索：#1634、#1647、#1704、#1710

近期解决方案可参考：https://github.com/the1812/Bilibili-Evolved/discussions/1634#discussioncomment-593169

**Reply by sherlocknieh** [reply]
我在 issue 区没找到相关问题，就直接新建了功能建议，开完才注意到是放在 discussion 区。
这是网站的引导没做好，不能怪用户呀。
这个问题不解决的话，以后还会有更多人不停的提重复问题，跟“善用搜索”无关。

**Reply by Geeyun-JY3** [reply]
开发者说了呀，ta 解决不了：https://github.com/the1812/Bilibili-Evolved/issues/1710#issuecomment-813303446
> 我救不了 跟b站反馈去吧
***
> 我在 issue 区没找到相关问题

上述重复的就有一个是 issue 哦。
***
> 这是网站的引导没做好

开 issue 前总要看看 [Issue 须知](https://github.com/the1812/Bilibili-Evolved/blob/preview/doc/issue-rules.md) 吧，[讨论区指南](#1297) 也有写：
> 善用搜索(界面汉化截图见下方), 请不要开重复的主题!
***
> 这个问题不解决的话，以后还会有更多人不停的提重复问题，跟“善用搜索”无关。

这不是开重复问题的借口。
之前都试过多少次了，反馈相同问题的 issue 在第一页或已经被置顶，甚至给出明确回复会解决，照样有人继续开重复的 issue，应该吗？

这是个开源项目，修 bug 也需要时间，就算 bug 对使用体验影响巨大，也不能苛求开发者光速 hotfix。
开发者对一个 issue 给出明确回复了，就不应该再开重复问题。

**Reply by the1812** [reply]
我改了下 issue 发起那边, 去掉了能直接发起 discussion 的链接, 太容易直接发起也是导致不去搜索的一个原因.

**Reply by Geeyun-JY3** [reply]
https://github.com/the1812/Bilibili-Evolved/issues/1076#issuecomment-718056560 前 车 之 鉴（
